src/public/app/widgets/type_widgets/image.ts

```diff
@@ -54,13 +54,16 @@ class ImageTypeWidget extends TypeWidget {
         this.$imageWrapper = this.$widget.find(".note-detail-image-wrapper");
         this.$imageView = this.$widget.find(".note-detail-image-view").attr("id", `image-view-${utils.randomString(10)}`);
 
-        libraryLoader.requireLibrary(libraryLoader.WHEEL_ZOOM).then(() => {
-            WZoom.create(`#${this.$imageView.attr("id")}`, {
-                maxScale: 50,
-                speed: 1.3,
-                zoomOnClick: false
-            });
-        });
+        setTimeout(() => {
```

Comment:
Why the timeout? Feels like a workaround, maybe we need to find the root cause.

If it's to not block the rendering process while initializing the WheelZoom, I would prefer an `async` method that is called without `await`.